src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -455,9 +456,18 @@ public function formAction($id, Request $request)
         }
 
         $productAdapter = $this->get('prestashop.adapter.data_provider.product');
-        $product = $productAdapter->getProduct($id);
+        try {
+            $product = $productAdapter->getProduct($id);
+        } catch (LogicException $e) {
```

Comment:
It seems LogicException is raised only if we don't provide a product id, but if we cant to call this route we have to give a product id, no ?

Reply:
@mflasquin sorry I don't understand your question